### PR TITLE
release-20.2: backupccl: change CREATE SCHEDULE output `name` column to `label`

### PR DESCRIPTION
**Pull request by adityamaru**
Backport 1/1 commits from #54353.

/cc @cockroachdb/release

---

This change makes the output of the CREATE SCHEDULE statement more
in line with SHOW SCHEDULES. Since #53129 a scheduled job has a
"label" not a "name" to establish more accurate user expectations.

Fixes: #54210

Release note (sql change): CREATE SCHEDULE output now has a column
called `label` which was previously called `name`.

### CODE REVIEW

**Review by dt — APPROVED** [review]
just need to wait on approval over on #53662 but looks very low-risk to me

**Comment by adityamaru** [comment]
merging this and adding it to the google sheet since the release tracking process has changed.

**Pull request merged**
